src/components/timeline/ActiveTabGlobalTrack.js

```diff
@@ -0,0 +1,165 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import React, { PureComponent } from 'react';
+import classNames from 'classnames';
+import { selectActiveTabTrack } from '../../actions/profile-view';
+import {
+  getSelectedThreadIndex,
+  getSelectedTab,
+} from '../../selectors/url-state';
+import explicitConnect from '../../utils/connect';
+import { getActiveTabGlobalTracks } from '../../selectors/profile';
+import './Track.css';
+import TimelineTrackThread from './TrackThread';
+import TimelineTrackScreenshots from './TrackScreenshots';
+import { assertExhaustiveCheck } from '../../utils/flow';
+
+import type { TabSlug } from '../../app-logic/tabs-handling';
+import type { GlobalTrackReference } from '../../types/actions';
+import type {
+  TrackIndex,
+  ActiveTabGlobalTrack,
+  InitialSelectedTrackReference,
+} from '../../types/profile-derived';
+import type { ConnectedProps } from '../../utils/connect';
+
+type OwnProps = {|
+  +trackReference: GlobalTrackReference,
+  +trackIndex: TrackIndex,
+  +style?: Object /* This is used by Reorderable */,
+  +setInitialSelected: (el: InitialSelectedTrackReference) => void,
+|};
+
+type StateProps = {|
+  +globalTrack: ActiveTabGlobalTrack,
+  +isSelected: boolean,
+  +selectedTab: TabSlug,
+|};
+
+type DispatchProps = {|
+  +selectActiveTabTrack: typeof selectActiveTabTrack,
+|};
+
+type Props = ConnectedProps<OwnProps, StateProps, DispatchProps>;
+
+class ActiveTabGlobalTrackComponent extends PureComponent<Props> {
```

Comment:
This should have some documentation for this component, and what the intent is for it.